ebl/v3/EBL_v3.0.0.yaml

```diff
@@ -4437,6 +4437,8 @@ components:
             example: Strawinskylaan 4117
         identifyingCodes:
           type: array
+          description: |
```

Comment:
Shouldn't we change this to one of?

Reply:
In this case it is ok to provide both (it helps identifying the party)